scripts/validate_docstrings.py

```diff
@@ -455,6 +455,12 @@ def validate_one(func_name):
             if not rel_desc:
                 errs.append('Missing description for '
                             'See Also "{}" reference'.format(rel_name))
+
+    for line in doc.raw_doc.splitlines():
+        if re.match(" *\t", line):
```

Comment:
I think the regex would be
    `if re.match(r"^\t", line)`

Reply:
Indeed, missing the start of the line. Sorry about that. I was checking as well there is no spaces followed by tabs. Is it something we should check or just a plain "starts with a tab"

Reply:
Ahh, I see. So maybe `^ *\t`? That's lines starting with 0 or more spaces followed by a tab.

Reply:
Do you want me to add a test file for this script? I can create a tests folder with a test for this script that will 1) prevent future mistakes and would have made easier to review a PR like this one. What do you think?

Reply:
Actually, There is already another PR adding the test file :)